valohai_yaml/__main__.py

```diff
@@ -4,31 +4,47 @@
 import sys
 from collections import defaultdict
 
+from valohai_yaml.lint import lint
 from .validation import validate
 
 
 def main(args=None):
     ap = argparse.ArgumentParser()
     ap.add_argument('file', nargs='+', help='file(s) to validate')
+    ap.add_argument('--lint', action="store_true", help='Use linter')
```

Comment:
I don't think this needs to be an option – the linter is a superset of the validator, so we can just as well always use it.

Reply:
Fix pushed